.github/ISSUE_TEMPLATE/bug-task.yaml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
```

Comment:
Good thinking, but probably not necessary, we'll see their GitHub username!

Reply:
Making a note here, that at this point I would agree, but in the future depending on the project size and org, sometimes the quickest contact method isn't via GH.